uploadNeuroDB/NeuroDB/ImagingUpload.pm

```diff
@@ -478,7 +486,7 @@ Arguments:
 sub runCommand {
     my $this = shift;
     my ($command) = @_;
-    print "\n\n $command \n\n " if $this->{'verbose'};
+    print "\n\n $command \n\n " if $this->{verbose};
```

Comment:
@mohadesz,

It only worked as expected as I removed the single quotation. If you check the MRIProcessing utility (for example lines 315 and 319 in computeMd5hash(), it seems to be used without single quotations as well). I am really not sure why.